brooklyn-server/rest/rest-server/src/main/java/org/apache/brooklyn/rest/resources/ServerResource.java

```diff
@@ -218,6 +218,7 @@ public void run() {
                         if (shutdownHandler != null) {
                             shutdownHandler.onShutdownRequest();
                         } else {
+                            // should always be set as it is required by jersey injection?
```

Comment:
I think some tests could inject null value if the hook is of no interest to them. Certainly shouldn't get to this piece of code in this case.
Probably safe to remove now.